test/vfd_swmr_group_writer.c

```diff
@@ -12,13 +12,13 @@
  */
 
 #include <err.h>
+#include <unistd.h> /* getopt(3) */
```

Comment:
unistd.h shouldn't be needed as it's included in H5private.h.

Reply:
I'll take it out.  I don't know how it slipped in.